public/js/show.js

```diff
@@ -0,0 +1,82 @@
+$(document).ready(function() {
+    $("#timeslot-list").hide();
```

Comment:
2 spaces seems to be used for JS elsewhere. You should make your editor pay attention to the .editorconfig :p